tools/ci/download_tar.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# */30 * * * * bash /data/app/cdn/download_tar.sh > /data/app/cdn/download.log 2>&1
+
+org=FISCO-BCOS
+repo=(FISCO-BCOS console)
+# output_dir=/data/app/cdn/
+output_dir=~/WorkSpace/WeBank/myfisco/build/cdn/
+timeout=300
+
+set -e
+
+failed_clean()
+{
+    local path=${1}
+    rm -rf ${path}
+    exit 1
+}
+
+download_fisco_artifact()
+{
+    local version=${1}
+    local from=https://github.com/FISCO-BCOS/FISCO-BCOS/releases/download/v${version}
+    local to=${output_dir}/fisco-bcos/releases/download/v${version}
+    mkdir -p ${to}
+    local tars=(fisco-bcos.tar.gz fisco-bcos-gm.tar.gz fisco-bcos-macOS.tar.gz)
+    for file in ${tars[*]}
+    do
+        curl -Lo ${to}/${file} ${from}/${file} -m ${timeout} || failed_clean ${to}
+    done
+}
+
+download_fisco()
+{
+    local latest_version=$(curl -s https://api.github.com/repos/FISCO-BCOS/FISCO-BCOS/releases | grep "tag_name" | grep "\"v2\.[0-9]\.[0-9]\"" | sort -u | tail -n 1 | cut -d \" -f 4 | sed "s/^[vV]//")
+    mkdir -p ${output_dir}/fisco-bcos/releases/download/
+    local local_latest=$(ls -1 ${output_dir}/fisco-bcos/releases/download/ | sort -u | tail -n 1)
+    echo "latest fisco-bcos is ${latest_version}, local is ${local_latest}"
+    if [ "v${latest_version}" != "${local_latest}" ];then
+        echo "download fisco-bcos ${latest_version} ..."
+        download_fisco_artifact ${latest_version}
+    else
+        echo "fisco-bcos ${latest_version} tars already have."
+    fi
+}
+
+download_console()
+{
+    local latest_version=$(curl -s https://api.github.com/repos/FISCO-BCOS/console/releases/latest | grep "tag_name" | sort -u | tail -n 1 | cut -d \" -f 4 | sed "s/^[vV]//")
+    mkdir -p ${output_dir}/console/releases/download/
+    local local_latest=$(ls -1 ${output_dir}/console/releases/download/ | sort -u | tail -n 1)
+    echo "latest console is ${latest_version}, local is ${local_latest}"
+    local dist_dir=${output_dir}/console/releases/download/v${latest_version}
+    if [ "v${latest_version}" != "${local_latest}" ];then
+        echo "download console ${latest_version} ..."
+        mkdir -p ${dist_dir}
+        local console_timeout=$(( timeout * 2 ))
```

Comment:
I think the `console_timeout` is a little small, you can test this value when deploy the shell script.

Reply:
10 mins are large, but you are right, we can test.